pinc/user_project_info.inc

```diff
@@ -19,28 +19,37 @@ various places:
 
 function upi_set_t_latest_home_visit( $username, $projectid, $timestamp )
 {
-    mysqli_query(DPDatabase::get_connection(), "
+    $sql = sprintf("
         INSERT INTO user_project_info
         SET
-            username            = '$username',
-            projectid           = '$projectid',
-            t_latest_home_visit = $timestamp
+            username            = '%s',
+            projectid           = '%s',
+            t_latest_home_visit = %d,
```

Comment:
The comma here makes the SQL invalid (found after testing it in my sandbox).

Reply:
Sorry :(